### PR TITLE
[Issue - 107 ]Added Egg dropping Puzzle Solution in C

### PR DESCRIPTION
**Pull request by SaurabhVj**
# Added Recursive solution of Egg Dropping Puzzle in C language and updated the link of the solution in C README

Fixed Issue #107 
## Type of change

-   [x] New feature (non-breaking change which adds functionality)
-   [x] Update C README

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generates no new warnings
-   [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by YashSaxena9** [comment]
Fill the PR template properly and edit the README.md file in the C language folder. :)

**Comment by SaurabhVj** [comment]
> Fill the PR template properly and edit the README.md file in the C language folder. :)

Okay

File: C/EggDroppingPuzzle.c [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/* Saurabh Vijay
+   Github :- https://github.com/SaurabhVj
+   Solution of Egg Dropping Puzzle
+*/
+
+#include<stdio.h>
+#define INT_MAX 1e9+9 
+
+int maxi(int a , int b)
```
Comment (kaustubhgupta) [inline C/EggDroppingPuzzle.c]: use better function name

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@SaurabhVj edit the PR template to fill all the required details, most importantly mention issue number which this PR fixes

**Comment by SaurabhVj** [comment]
> @SaurabhVj edit the PR template to fill all the required details, most importantly mention issue number which this PR fixes

Done that @kaustubhgupta

**Comment by kaustubhgupta** [comment]
I cannot see the changes. You have deleted the fixes line from the template. Mention it again as "Fixes #(issue number)"

**Comment by SaurabhVj** [comment]
> I cannot see the changes. You have deleted the fixes line from the template. Mention it again as "Fixes #(issue number)"

@kaustubhgupta Can you see the changes now ?

**Comment by kaustubhgupta** [comment]
@SaurabhVj thanks for contributing! These are necessary changes so that we can keep a track of that issues have been solved!

**Pull request merged**